ocaml/idl/datamodel.ml

```diff
@@ -5173,6 +5173,9 @@ module VM_guest_metrics = struct
         ; field ~qualifier:DynamicRO
             ~ty:(Map (String, String))
             "os_version" "version of the OS"
+        ; field ~qualifier:DynamicRO
+            ~ty:(Map (String, String))
+            "host_name" "The NETBIOS name of the machine"
```

Comment:
Calling this `host_name` may be confusing, as there are other "host names" around in the API. Can we make it more specific?

Reply:
`win_fqdn`?

Reply:
`netbios_name`? `fqdn` is a very technical term.

Reply:
Renamed to netbios_name